lua/lib/renderer.lua

```diff
@@ -195,9 +195,14 @@ local function update_draw_data(tree, depth, markers)
         table.insert(lines, padding..icon..git_icon..node.name)
       end
     elseif node.link_to then
+      local icon = ""
+      if not icon_state.show_file_icon then
+        icon = icon_state.icons.default
+      else
```

Comment:
this would be bad for performance, could you make a function that is generated at first (like i did for folders and default icons). Although there shouldn't be too many symlinks, there actually could be (open vim in `/usr/lib` for instance),

Reply:
I'll try to see how to do this. BTW, shouldn't this be another option? Having an icon for symlinks only, not use the same for files.

Reply:
we could add this indeed. If you feel like it :)